doc/quick_start/contributor-guide.rst

```diff
@@ -75,25 +75,52 @@ Here are guidelines for the review process that you and the maintainers will fol
 #. The general commit message format is::
 
      <component>: <summary>
+
      <description>
+
      [Fixes|Related-to] OP-<number>
```

Comment:
Agreed, that would be better, because in practice we've always used the colon. Will update this.